integreat_cms/cms/models/offers/offer_template.py

```diff
@@ -58,6 +61,12 @@ class OfferTemplate(AbstractBaseModel):
             "Whether the Integreat app supports displaying offers from this provider in pages"
         ),
     )
+    is_zammad_form = models.BooleanField(
```

Comment:
Maybe we should be more specific here. While we use a generic Zammad form in the back end, the app will implement a specific set of fields for this. That means there could be other forms in the app that use Zammad as a back end for posting the content. We could rename it to `is_malte_zammad_form`. See https://github.com/digitalfabrik/integreat-app/issues/2316

Reply:
Hm. The `is_zammad_form` is currently only being used internally though, to determine if the offer can be embedded in a page, what other offer fields are required, and so on. The idea was that form specifics (like what fields are required, etc.) are passed on through the configurable post parameters in the offer template settings.

Unless I am missing something about how this is supposed to work, I'd rather keep this setting as-is, and simply create an OfferTemplate "Zammad Malte" with `supported_by_app_in_content` and `is_zammad_form` and required fields etc. configured in its parameters.